src/client/components/configuration/view.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react'
+import * as style from './style.css'
+import { Node, Tree } from './tree/tree'
+import { TreeStore } from './tree_store'
+
+const store = new TreeStore({
+    label: 'root',
+    expanded: true,
+    leaf: false,
+    selected: false,
+    children: [
+        {
+            label: 'parent',
+            expanded: false,
+            leaf: false,
+            selected: false,
+            children: [
+                { label: 'child1', expanded: false, leaf: true, selected: false },
+                { label: 'child2', expanded: false, leaf: true, selected: false },
+            ],
+        },
+        {
+            label: 'parent',
+            expanded: false,
+            leaf: false,
+            selected: false,
+            children: [
+                {
+                    label: 'nested parent',
+                    expanded: false,
+                    leaf: false,
+                    selected: false,
+                    children: [
+                        { label: 'nested child 1', expanded: false, leaf: true, selected: false },
+                        { label: 'nested child 2', expanded: false, leaf: true, selected: false },
+                    ],
+                },
+            ],
+        },
+        {
+            label: 'Some file here',
+            expanded: true,
+            leaf: true,
+            selected: false,
+            children: [] as Node[],
+        },
+    ],
+})
+
+export const Configuration = () => (
+  <div className={style.configuration}>
+    <div className={style.configuration__header}>
+      <h1 className={style.configuration__headerTitle}>Configuration</h1>
```

Comment:
I've made two wrapper components [view](https://github.com/NUbots/NUsight2/tree/feature/configuration/src/client/components/configuration/view) (for top-level views like dashboard or localization) and [panel](https://github.com/NUbots/NUsight2/tree/feature/configuration/src/client/components/configuration/panel) (for stuff like sidebars) which could be useful.

I've placed them under [`components/configuration`](https://github.com/NUbots/NUsight2/tree/feature/configuration/src/client/components/configuration) for now, but I think they should go higher (`components/view|panel/` or `components/common/view|panel/`?) since they could be used by other views.

I can extract them into a separate pull request if needed.